rbs/rbs_common.cc

```diff
@@ -2,11 +2,50 @@
 
 namespace sorbet::rbs {
 
-core::LocOffsets locFromRange(core::LocOffsets loc, const rbs_range_t &range) {
+core::LocOffsets RBSDeclaration::commentLoc() const {
     return {
-        loc.beginPos() + range.start.char_pos,
-        loc.beginPos() + range.end.char_pos,
+        comments.front().commentLoc.beginLoc,
+        comments.back().commentLoc.endLoc,
     };
 }
```

Comment:
You can simplify this into one line with

```cpp
return comments.front().commentLoc.join(comments.back().commentLoc);
```

not blocking

Reply:
Fixed in https://github.com/sorbet/sorbet/pull/8760.